README.md

```diff
@@ -1,3 +1,17 @@
+# :warning: Archiving Note
+
+This repository is no longer being maintained and has been archived for historical purposes. 
+
+We have now developed [DeepRank2](https://github.com/DeepRank/deeprank2), an improved and unified version of DeepRank, [DeepRank-GNN](https://github.com/DeepRank/Deeprank-GNN), and [DeepRank-Mut](https://github.com/DeepRank/DeepRank-Mut).
+
+:sparkles: DeepRank2 allows for transformation and storage of 3D representations of both protein-protein interfaces (PPIs) and protein single-residue variants (SRVs) into either graphs or volumetric grids containing structural and physico-chemical information. These can be used for training neural networks for a variety of patterns of interest, using either our pre-implemented training pipeline for graph neural networks (GNNs) or convolutional neural networks (CNNs) or external pipelines.
+
+- :wrench: **Pull Requests** at github.com/DeepRank/deeprank2
+- :bug: **Bugs**: Reports of bugs can be filed agains our new repo github.com/DeepRank/deeprank2
+- :star: **Feature Requests**: Add your request or discuss the project w/ the community at github.com/DeepRank/deeprank2
+
```

Comment:
```suggestion
```

Reply:
It's also ok to keep them, but please preview the links and make sure they look good